### PR TITLE
Use server id

### PR DESCRIPTION
**Pull request by BeaudanBrown**
Revert the sending of message UUIDs from earlier and instead add a server id field to messages which is only used for public chats
This will allow us to send delete requests for a particular server/channel/messageServerId and have all the clients know which message this is for

### CODE REVIEW

**Comment by BeaudanBrown** [comment]
Server id is just an autoinc int on a per-channel basis, so for a deletion request we will be essentially looking for a message in the public channel conversation with a particular serverId (e.g. 53)

File: js/models/conversations.js [hunk 1]
```diff
@@ -2051,6 +2051,20 @@
...
       return endpoint;
     },
+    getChannelToken() {
+      if (!this.isPublic()) {
+        return null;
+      }
+      return this.get('channelToken');
+    },
+    async setChannelToken(token) {
+      if (!_.isEqual(this.get('channelToken'), token)) {
```
Comment (sachaaaaa) [inline js/models/conversations.js]: Just curious, what does `_.isEqual` do differently than `==`/`===` ?

Comment (sachaaaaa) [reply js/models/conversations.js]: Oh is it because `token` is array-like and every item needs to be compared one by one?

**Comment by sachaaaaa** [comment]
the tests seem to fail due to the linter.

**Comment by BeaudanBrown** [comment]
Whoops that last commit was meant to be pushed to another branch

**Pull request merged**